ProjectProtocol/project-protocol-web
# [PRO-314] Bug fix: password reset form strings not displaying properly

This page not only had broken UI strings, but was also crashing because of the use of the PageHeader component in BasicPage, which needs to be rendered within the LoginUIProvider, which this page (and the confirmations page) are not.

Changes
---
* Make separate FullscreenLayout for these screens so BasicPage can continue to use PageHeader.
* Update translations and keys to fix UI

Testing
---
1. Go to login > forgot password? > reset password with an account you have email access to (it's worth creating a personal account on staging for this purpose. Note that deploy previews use the staging back-end, so you can do this all in the preview).
2. Copy the "Reset Password" link in the email and update the first section to be the preview domain and paste in your browser: 
```
Original link:

https://ratemypo-staging.netlify.app/password_resets/<token>

Alter and paste into browser

https://deploy-preview-125--fabulous-sprite-a61e48.netlify.app/password_resets/<token>
```
3. See that the page renders correctly in English and Spanish
4. See that password change is successful.

Notes
---
I tried to make this fix with a minimum of file changes, although I see two pieces of tech debt we should address at some point:
1. Our translations should be organized and separated into a namespace-per-page structure so we can more easily navigate and compare them. We have a sizable amount of duplicates and much too much nesting going on in these files today. [Namespaces](https://react.i18next.com/latest/usetranslation-hook#loading-namespaces).
2. These pages should be rendered within the normal app layout so the user has context and ability to go to the main pages at any time.
